dm/dm.go

```diff
@@ -50,6 +50,8 @@ var (
 	username          = flag.String("username", "", "Github user name that overrides GITHUB_USERNAME environment variable")
 	password          = flag.String("password", "", "Github password that overrides GITHUB_PASSWORD environment variable")
 	apitoken          = flag.String("apitoken", "", "Github api token that overrides GITHUB_API_TOKEN environment variable")
+	serviceaccount    = flag.String("serviceaccount", "", "Service account file containing JWT token")
+	registryfile      = flag.String("registryfile", "", "File containing registry specification")
 )
 
 var commands = []string{
```

Comment:
I'm assuming that @adamreese, et. al. will be pulling bits from this client to that one, but not mission critical.